### PR TITLE
monitor(engine): add basic Grafana panel for tiflow

### LINKED ISSUE #6995: Add basic grafana page for Tiflow
As the title said.

### PR DESCRIPTION
**Pull request by maxshuang**
### What problem does this PR solve?

Issue Number: close #6995 

### What is changed and how it works?
Add basic Grafana panel for tiflow

#### Tests 
 - Manual test (add detailed scripts or steps below)
 
image

### Release note 

```release-note
 `None`.
```

### CODE REVIEW

**Comment by gozssky** [comment]
Can we put `tiflow.json` inside https://github.com/pingcap/tiflow/tree/master/metrics/grafana? It seems no conflicts.

**Review by gozssky — APPROVED** [review]

File: metrics/grafana/tiflow.json [hunk 1]
```diff
@@ -0,0 +1,766 @@
...
+                "uid": "zx0TiSM4k"
+              },
+              "editorMode": "code",
+              "expr": "max(dataflow_dataflow_server_master_job_num{}) by (status)",
+              "legendFormat": "{{status}}",
+              "range": true,
+              "refId": "A"
+            }
+          ],
+          "title": "Job Num",
```
Comment (lonng) [inline metrics/grafana/tiflow.json:131]: The panel is not about job numbers. It's a job status stats.

File: metrics/grafana/tiflow.json [hunk 2]
```diff
@@ -0,0 +1,766 @@
...
+              },
+              "editorMode": "code",
+              "expr": "max(dataflow_dataflow_server_master_executor_num{}) by (status)",
+              "interval": "1",
+              "legendFormat": "{{status}}",
+              "range": true,
+              "refId": "A"
+            }
+          ],
+          "title": "Executor Num",
```
Comment (lonng) [inline metrics/grafana/tiflow.json:223]: ditto

File: metrics/grafana/tiflow.json [hunk 3]
```diff
@@ -0,0 +1,766 @@
...
+        "x": 0,
+        "y": 0
+      },
+      "id": 12,
+      "panels": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "zx0TiSM4k"
+          },
```
Comment (amyangfei) [inline metrics/grafana/tiflow.json:46]: The `uid` seems not portable, should we add some datasource input. ref https://github.com/pingcap/tiflow/blob/4a97c4527a56d531d0abd9f9f599ac9006dc8275/metrics/grafana/ticdc.json#L2-L11

File: metrics/grafana/tiflow.json (hunk 1 above)
Comment (maxshuang) [reply metrics/grafana/tiflow.json]: I am a little struggle. 
Seems that `Job State Stats` is  too general.
`Job State Num` and `Num of Job State` sounds weird too.

Comment (maxshuang) [reply metrics/grafana/tiflow.json]: Replace `Job Num` with `Num of Job State`

File: metrics/grafana/tiflow.json (hunk 2 above)
Comment (maxshuang) [reply metrics/grafana/tiflow.json]: replace `Executor Num` with `Num of Executor state`

File: metrics/grafana/tiflow.json (hunk 3 above)
Comment (maxshuang) [reply metrics/grafana/tiflow.json]: Using externel datasource when output json.

**Review by amyangfei — APPROVED** [review]

**Comment by amyangfei** [comment]
Unit test failed @maxshuang 

ref: https://ci.pingcap.net/blue/organizations/jenkins/ut-check/detail/ut-check/33127/pipeline

```go

[2022-09-07T11:55:25.326Z] === Failed

[2022-09-07T11:55:25.326Z] === FAIL: engine/servermaster TestCollectMetric (1.00s)

[2022-09-07T11:55:25.326Z]     server_test.go:297: 

[2022-09-07T11:55:25.326Z]         	Error Trace:	/home/jenkins/agent/workspace/ut-check/tiflow/engine/servermaster/server_test.go:297

[2022-09-07T11:55:25.326Z]         	            				/home/jenkins/agent/workspace/ut-check/tiflow/engine/servermaster/server_test.go:289

[2022-09-07T11:55:25.326Z]         	Error:      	Condition never satisfied

[2022-09-07T11:55:25.326Z]         	Test:       	TestCollectMetric

[2022-09-07T11:55:25.326Z] 

[2022-09-07T11:55:25.326Z] DONE 399 tests, 1 failure in 332.797s
```

**Comment by maxshuang** [comment]
> Unit test failed @maxshuang
> 
> ref: https://ci.pingcap.net/blue/organizations/jenkins/ut-check/detail/ut-check/33127/pipeline
> 
> ```go
> [2022-09-07T11:55:25.326Z] === Failed
> 
> [2022-09-07T11:55:25.326Z] === FAIL: engine/servermaster TestCollectMetric (1.00s)
> 
> [2022-09-07T11:55:25.326Z]     server_test.go:297: 
> 
> [2022-09-07T11:55:25.326Z]         	Error Trace:	/home/jenkins/agent/workspace/ut-check/tiflow/engine/servermaster/server_test.go:297
> 
> [2022-09-07T11:55:25.326Z]         	            				/home/jenkins/agent/workspace/ut-check/tiflow/engine/servermaster/server_test.go:289
> 
> [2022-09-07T11:55:25.326Z]         	Error:      	Condition never satisfied
> 
> [2022-09-07T11:55:25.326Z]         	Test:       	TestCollectMetric
> 
> [2022-09-07T11:55:25.326Z] 
> 
> [2022-09-07T11:55:25.326Z] DONE 399 tests, 1 failure in 332.797s
> ```

I will fix it.

**Pull request merged**